avg_pool2d_backward test case is broken.

### Describe the bug

The `avg_pool2d_backward` benchmark always fail with following information:

```
FAILED benchmark/test_avg_pool2d.py::test_avg_pool2d_backward - Failed: Overloaded torch operator invoked from Python failed to match any schema:
aten::avg_pool2d_backward() is missing value for argument 'self'. Declaration: aten::avg_pool2d_backward(Tensor grad_output, Tensor self, int[2] kernel_size, int[2] stride, int[2] padding, bool ceil_mode, bool count_include_pad, int? divisor_override) -> Tensor
aten::avg_pool2d_backward() is missing value for argument 'self'. Declaration: aten::avg_pool2d_backward.grad_input(Tensor grad_output, Tensor self, int[2] kernel_size, int[2] stride, int[2] padding, bool ceil_mode, bool count_include_pad, int? divisor_override, *, Tensor(a!) grad_input) -> Tensor(a!)

```

--- a/benchmark/test_avg_pool2d.py
+++ b/benchmark/test_avg_pool2d.py
@@ -79,18 +79,17 @@ def test_avg_pool2d():
     bench.run()
 
 
-@pytest.mark.skip(reason="Test case fails due to missing parameter self.")
 @pytest.mark.avg_pool2d_backward
 def test_avg_pool2d_backward():
     if flag_gems.vendor_name == "mthreads":
         dtypes = [torch.float32]
     else:
-        dtypes = [consts.FLOAT_DTYPES]
+        dtypes = consts.FLOAT_DTYPES
 
     bench = AvgPool2dBenchmark(
         input_fn=avg_pool2d_input_fn,
         op_name="avg_pool2d_backward",
-        torch_op=torch.ops.aten.avg_pool2d_backward,
+        torch_op=torch.ops.aten.avg_pool2d,
         dtypes=dtypes,
         is_backward=True,
     )

